src/DDTrace/Transport/Http.php

```diff
@@ -82,6 +82,11 @@ private function configure($config)
      */
     public function send(Tracer $tracer)
     {
+        if (function_exists('dd_tracer_circuit_breaker_can_try') && !dd_tracer_circuit_breaker_can_try()) {
```

Comment:
I'm guessing the `function_exists()` checks are just in case manually instrumented code from Composer gets out of sync with the installed extension? :)

Reply:
Actually yes :) 
I think we need to address this somehow more generally the more functions we'll add. For now since these are called only once per request its relatively fine - although definitely not pretty :|